acl_loader/main.py

```diff
@@ -290,6 +290,14 @@ def is_table_mirror(self, tname):
         """
         return self.tables_db_info[tname]['type'].upper().startswith(self.ACL_TABLE_TYPE_MIRROR)
 
+    def is_table_ipv6(self, tname):
+        """
+        Check if ACL table type is IPv6 (L3V6 or MIRRORV6)
+        :param tname: ACL table name
+        :return: True if table type is IPv6 else False
+        """
+        return "V6" in self.tables_db_info[tname]["type"].upper()
```

Comment:
>"V6" [](start = 15, length = 4)

Determine from table name seems not robust. Is there a better way?

Reply:
This is based on the table type, not the name.

Reply:
If my understanding is correct, the table types are from
[https://github.com/Azure/sonic-buildimage/blob/95ec75e24ed2c95186c9ab8367bc17dfadc0d44b/src/sonic-config-engine/minigraph.py#L450-L455](https://github.com/Azure/sonic-buildimage/blob/95ec75e24ed2c95186c9ab8367bc17dfadc0d44b/src/sonic-config-engine/minigraph.py#L450-L455)

Is it better to check `L3V6` or ends with `V6` to determine icmp/icmpv6?

---
In reply to: [584383685](https://github.com/Azure/sonic-utilities/pull/1469#discussion_r584383685) [](ancestors = 584383685)

Reply:
Ack, I updated it to check for a specific list of table types (in case we add some new table in the future that is IPv6 but does not have V6 as a suffix).